rspirv/sr/context.rs

```diff
@@ -16,18 +16,22 @@ use std::{
     collections::HashMap,
     hash::BuildHasherDefault,
     marker::PhantomData,
+    num::NonZeroU32,
+    ops,
 };
 
 use fxhash::FxHasher;
+use spirv;
 
 use crate::{
-    spirv,
     sr::constants::{Constant, ConstantEnum},
     sr::types::{StructMember, TypeEnum, Type},
 };
 
 type FastHashMap<K, V> = HashMap<K, V, BuildHasherDefault<FxHasher>>;
-/// An index corresponding to `Id` in SPIR-V.
+/// An unique index in the storage array that a token points to.
+///
+/// This type is independent of `spirv::Word`.
```

Comment:
More on the difference? What about:

This type is independent of `spirv::Word`. `spirv::Word` is used in data representation. It holds a SPIR-V <id> and refers to that instruction. In structured representation, we use `Token` to refer to an SPIR-V instruction. `Index` is an implementation detail to `Token`.